src/Symfony/Bundle/WebProfilerBundle/Resources/config/profiler.xml

```diff
@@ -27,7 +27,13 @@
             <argument type="service" id="twig" />
             <argument>%kernel.debug%</argument>
             <argument type="service" id="debug.file_link_formatter" />
-            <argument type="service" id="error_renderer.renderer.html" on-invalid="null" />
+            <argument type="service" id="error_renderer.renderer.html" />
```

Comment:
why removing `on-invalid="null"` ?

Reply:
It's a hard dependency on the bundle since 4.4, is there any chance it's invalid?